graspologic/embed/omni.py

```diff
@@ -25,6 +65,9 @@ def _get_omni_matrix(graphs):
     out : 2d-array
         Array of shape (n_vertices * n_graphs, n_vertices * n_graphs)
     """
+    if isspmatrix_csr(graphs[0]):
```

Comment:
Just adding this as a separate call so the numpy wizardry below can keep rockin out.